raster-foundry/raster-foundry
# Create a licenses table and a license column in datasources table

## Overview

This PR adds a migration to create a licenses table and a license column in datasources table.

### Checklist

~- [ ] Styleguide updated, if necessary~
~- [ ] Swagger specification updated, if necessary~

### Notes

This PR should be merged after Alex's PR #2947


## Testing Instructions

 * Check if a new licenses table is created with the following fields: short_name (primary key), name, url, osi_approved.
* Check if a new column called license_name is created in the datasources table referencing the licenses table.

Closes #2954
